env

```diff
@@ -0,0 +1,4 @@
+CLIENT_ID = "b7d43ed648454f8586d466bec5fc250b"
```

Comment:
This looks like a file that shouldn't be checked into version control for security reasons (you'll also need to change these credentials now that they've been made public). Please look at a way to define these locally and read into the project without committing them, and add instructions for how to set this up in the README.

Reply:
Comment implemented.
Added .env file in gitignore.